src/src/PluginStructs/P036_data_struct.cpp

```diff
@@ -370,20 +389,20 @@ void P036_data_struct::display_logo() {
   int top;
   tFontSettings iFontsettings = CalculateFontSettings(2); // get font with max. height for displaying "ESP Easy"
 
-  bDisplayingLogo = true;                                 // next time the display must be cleared completely
+  bDisplayingLogo = true; // next time the display must be cleared completely
   display->setTextAlignment(TEXT_ALIGN_LEFT);
   display->setFont(iFontsettings.fontData);
-  display->clear();                                       // resets all pixels to black
+  display->clear(); // resets all pixels to black
   display->setColor(WHITE);
-  display->drawString(65, iFontsettings.Top + TopLineOffset,                                              F("ESP"));
+  display->drawString(65, iFontsettings.Top + TopLineOffset, F("ESP"));
   display->drawString(65, iFontsettings.Top + iFontsettings.Height + iFontsettings.Space + TopLineOffset, F("Easy"));
 
   if (getDisplaySizeSettings(disp_resolution).PixLeft < left) { left = getDisplaySizeSettings(disp_resolution).PixLeft; }
-  top = (getDisplaySizeSettings(disp_resolution).Height - espeasy_logo_height) / 2;
+  top = (getDisplaySizeSettings(disp_resolution).Height-espeasy_logo_height)/2;
 
```

Comment:
That _was_ formatted using Uncrustify, mostly for better readability 😅

Reply:
I installed Uncrustfy in my VS Code GUI but it doesn't change the formatting.
Anything else to consider?

Reply:
Did you also install the Uncrustify executable, and pointed the plugin at it? (It can be a bit tricky to get it going at first) Then you can select it as the default formatter (Alt-Shift-F) for VS Code.

Reply:
I installed the Uncrustify extension and set it to my default Formatter.
But if I check the messages I got following information:
![grafik](https://user-images.githubusercontent.com/55511593/161625280-cb5a84eb-28a8-40a9-b097-ffb687ddc8eb.png)

Reply:
Open the settings (Ctrl-,) and search for 'uncrustify', then adjust the executable path where you unpacked the [separately downloaded](https://sourceforge.net/projects/uncrustify/) uncrustify executable (yep, I'm on Windows, and temporarily using the Light+ theme 😛).

![image](https://user-images.githubusercontent.com/1922393/161628148-c87587b6-915a-4764-ae1b-0aa0d9b2758b.png)

Reply:
Found it and it worked, see commit `Uncrustify code `